docs/nodejs: A note about building apps with C/C++ ext (#47)
paketo-buildpacks/paketo-website: content/docs/buildpacks/nodejs.md

If no package manager is detected, the Node.js CNB will set the start command
`node server.js`. The app name is ___not___ currently configurable.

## Stack support

> For most apps, the Node.js Buildpack runs fine on the Base builder. But when the app

Reviewer: ```suggestion
The Node.js Buildpack runs fine on the Base builder for most apps. If your app
```

requires compilation of native extensions using `node-gyp`, the buildpack requires that

> you use the Full builder. This is because `node-gyp` requires `python` that's absent on

Reviewer: ```suggestion
you use the Full builder. This is because `node-gyp` requires `python` which is excluded from the
```

> the Base builder, and the module may require other shared objects.

Reviewer: ```suggestion
the Base builder.
```

Author: I'm concerned not only about the requirements of `node-gyp` but about the requirements of the native module as such. Seems like base doesn't have many `lib*.so`s that full has.

Reviewer: oh oops, I didn't mean to delete that. Looks good now
